test/cmd/dnshammer.go

```diff
@@ -127,3 +165,44 @@ func lookupARecord(addr, resolverProtocol, domain string) ([]net.IP, error) {
 	defer cancel()
 	return resolver.LookupIP(ctx, "ip4", domain)
 }
+
+func lookupTXT(domain string) ([]string, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), lookupTimeout)
+	defer cancel()
+	return net.DefaultResolver.LookupTXT(ctx, domain)
+}
+
+func loadRecords(p string) map[string][]string {
```

Comment:
Why do we load them from disk, instead of just generating them here (and never hitting disk)?

Reply:
There is a disconnect between the test runner and the `dnsHammer` which is a standalone binary. The test generates the data in memory and writes it to the file for the `dnsHammer` to consume.

Reply:
Right, but _why_ does the test generate the test data, instead of letting `dnsHammer` do it?

Reply:
… because it needs in both the server and dnshammer.